common/src/main/java/com/wynntils/models/gear/GearModel.java

```diff
@@ -100,6 +100,16 @@ public CraftedGearItem parseCraftedGearItem(ItemStack itemStack) {
         WynnItemParseResult result = WynnItemParser.parseItemStack(itemStack, null);
         CappedValue durability = new CappedValue(result.durabilityCurrent(), result.durabilityMax());
         GearType gearType = GearType.fromItemStack(itemStack);
+        if (gearType == null) {
+            // If it is crafted, and has a skin, then we cannot determine weapon type from item stack
+            // Maybe it is possible to find in the string type, e.g. "Crafted Wand"
+            gearType = GearType.fromString(result.itemType());
+            if (gearType == null) {
+                // ... but if the item is signed, this will not work either. We're out of luck,
```

Comment:
I am confident skins only apply in appropriate slots. So armor won't have skins in your hand. I think..

Reply:
Yea, looks like this is the case in-game